### ISSUE TITLE
Update models and visualizers according to figma schemas

### ISSUE THREAD

**Issue by thiagogquinto** [activist-org/activist#744]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/activist-org/activist/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

The [figma schemas](https://www.figma.com/file/I9McFfaLu1RiiWp5IP3YjE/activist_public_designs?type=design&node-id=5377-33014&mode=design&t=PzjgvzFnDcuQIjbv-0) of some models have been updated with some new fields, this issue focuses on implementing the changes to the backend so the models and serializers be on the date.

### Contribution

Happy to work on this!

**Comment by andrewtavis** [comment]
Thanks @thiagogquinto! :)

**Comment by to-sta** [comment]
Hi @thiagogquinto, this might be helpful [graph_models](https://django-extensions.readthedocs.io/en/latest/graph_models.html#example-usage). This is part of the django extensions package and let's you create ER-Modell automatically based on the django-project. Haven't used it yet myself but thought this might be useful in the future.

**Comment by thiagogquinto** [comment]
@andrewtavis, the current _support_ table [here](https://github.com/activist-org/activist/tree/main/backend/authentication), has the following code:

```python
class Support(models.Model):
    supporter_type = models.ForeignKey(
        "SupportEntityType", on_delete=models.CASCADE, related_name="supporter"
    )
    supporter_entity = models.ForeignKey(
        "entities.Organization", on_delete=models.CASCADE, related_name="supporter"
    )
    supported_type = models.ForeignKey(
        "SupportEntityType", on_delete=models.CASCADE, related_name="supported"
    )
    supported_entity = models.ForeignKey(
        "entities.Organization", on_delete=models.CASCADE, related_name="supported"
    )

    def __str__(self) -> str:
        return f"{self.id}"
```

But the figma schema show that the fields ```supporter_entity``` and ```supported_entity``` points to User's ID, by the name looks like that the current code is correct, but I want to be sure.

**Comment by thiagogquinto** [comment]
@andrewtavis , some questions have come up, would you be able to have a meeting sometime?

**Comment by andrewtavis** [comment]
Yes sure, but it'd likely be best if @to-sta is there as well. He and I are meeting tonight at 18:00 UTC. I would that work for you too?

**Comment by thiagogquinto** [comment]
Unfortunetely I won't be available at that time, I will be avaluable only after 19:30 UTC, or on thurday and friday from 16:50 to 18:30 UTC for week days. Would some of this times work for ypu too?

**Comment by andrewtavis** [comment]
I'm on vacation starting tomorrow, so I'm not sure on it. Could you make the dev sync on Saturday?

**Comment by thiagogquinto** [comment]
Sorry for the late response. Maybe I can, I try my best to be there!

**Comment by thiagogquinto** [comment]
Actually I think I can try to explain here, to update a model I need to add the tables [status](https://www.figma.com/file/I9McFfaLu1RiiWp5IP3YjE/activist_public_designs?type=design&node-id=8641-13709&mode=design&t=4lhfJ523ZEhqteWo-4) and [status_entity_type](https://www.figma.com/file/I9McFfaLu1RiiWp5IP3YjE/activist_public_designs?type=design&node-id=8641-13709&mode=design&t=4lhfJ523ZEhqteWo-4).

I said that I would not add tables, but I am asking for this case because one of the already existing tables would need this table, so I would like to know if I may add this tables. I already had a general check and this would be the only tables added on this issue. The other tables would be added in the issues that already exist and also in the future issues that will be created.

**Comment by thiagogquinto** [comment]
For the Image, at least for the start idea, what do you think in adding checks for file type and size of the image?

**Comment by andrewtavis** [comment]
Sounds good, @thiagogquinto! #670 has a full rundown of the ideas we've had so far for image validation. Anything you feel comfortable adding from there would be much appreciated!

**Comment by andrewtavis** [comment]
Closed via #757 🚀 Thanks so much, @thiagogquinto!
